### PR TITLE
rebuild script

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "install": "node ./preinstall.js && node-gyp rebuild",
     "build": "node-gyp rebuild",
+    "rebuild": "rm -rf node_modules && npm cache clean --force && npm install",
```
Comment (dmarcos) [inline package.json]: maybe `clean` is a better name?

Comment (dmarcos) [reply package.json]: also `rm` does not work on windows

File: package.json [hunk 2]
```diff
@@ -62,6 +63,7 @@
     "puppeteer": "^1.5.0",
     "repl.history": "^0.1.4",
     "rimraf": "^2.6.2",
+    "shx": "^0.3.1",
```
Comment (modulesio) [inline package.json]: devDependencies?

**Comment by modulesio** [comment]
👍

**Pull request merged**